docs/config/lua/config/skip_close_confirmation_for_processes_named.md

```diff
@@ -50,6 +50,28 @@ return {
 
 *Since: 20220101-133340-7edc5b5a*
 
+For closing WSL Terminals on Windows without confirmation, the (non default) processes `wslhost.exe`, `wsl.exe`,
```

Comment:
1. The placement of this paragraph relative to the *Since* text above is incorrect; I think you probably want this text to go above it, as the *Since* stuff applies to the `mux-is-process-stateful` event mentioned below.

2.  If you're running `vim` inside the WSL instance in one of those tabs, does your suggested configuration cause it to be killed without prompting? If so, I don't want to make that suggestion in the docs as it can result in lost work in those programs. It would need to point out that it is a "dangerous" setting to employ.

Reply:
I tested vim and it closes without warning. Therefore, I included a section to point out the potential dangers as well as moving the _Since ..._ paragraph down.